packages/arb-token-bridge-ui/src/components/TokenModal/TokenModal.tsx

```diff
@@ -208,91 +241,120 @@ export const TokenListBody = () => {
     </div>
   )
 }
-export const TokenModalBody = ({
-  onTokenSelected,
-  toggleCurrentPannel
-}: {
-  onTokenSelected: () => void
-  toggleCurrentPannel: () => void
-}): JSX.Element => {
-  const [confirmationOpen, setConfirmationOpen] = useState(false)
-  const [blacklistedOpen, setBlacklistedOpen] = useState(false)
 
+function toERC20BridgeToken(data: L1TokenData): ERC20BridgeToken {
+  return {
+    name: data.name,
+    type: TokenType.ERC20,
+    symbol: data.symbol,
+    address: data.contract.address,
+    decimals: data.decimals
+  }
+}
+
+const SpecialETHAddress = 'eth.address'
```

Comment:
Took me a bit to see how this was working, but looks like it makes sense; why don't use the CONSTANT_CASE convention, and maybe name it something like ETH_IDENTIFIER? idk